include/ck/tensor_operation/gpu/grid/gridwise_elementwise_layernorm_welford_variance.hpp

```diff
@@ -0,0 +1,499 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck/utility/data_type.hpp"
+#include "ck/tensor_operation/gpu/element/binary_element_wise_operation.hpp"
+#include "ck/tensor_operation/gpu/block/blockwise_welford.hpp"
+#include "ck/tensor_operation/gpu/thread/threadwise_welford.hpp"
+#include "ck/tensor_operation/gpu/thread/threadwise_tensor_slice_transfer.hpp"
+#include "ck/tensor_operation/gpu/element/element_wise_operation.hpp"
+
+namespace ck {
+
+// Y = LayerNorm(A + B, Beta, Gamma)

```

Comment:
I thought the naming of ElementwiseOperation and AccElementwiseOperation is confusing.
device_gemm is a good example to imitate.
I suggest to rewrite the comment like this
X = A +B
Y = Layernorm(X, beta, gamme)
Therefore, you can rename 
ElementwiseOperation -> XElementwiseOperation
AccElementwiseOperation-> YElementwiseOperation